Research/lva-node-app/eventhubreader.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * This file contains the class used to get and process iot hub messages via Event Hub
+ * @fileoverview
+ * @requires module:event-hubs 
+ * @requires module:iot-hub-connection-string
+ */
+
+const { EventHubConsumerClient } = require('@azure/event-hubs');
+const { convertIotHubToEventHubsConnectionString } = require('./iot-hub-connection-string.js');
+
+/**
+* get messages coming from IoT Hub
+* @class
+*/
+class EventHubReader 
+{
+
+  /**
+   * @constructor
+   * @param {string} iotHubConnectionString - iot hub connection string in standard format
+   * @param {string} consumerGroup - consumer group, typically Default
+   */
+  constructor(iotHubConnectionString, consumerGroup) 
+  {
+    this.iotHubConnectionString = iotHubConnectionString;
+    this.consumerGroup = consumerGroup;
+  }
+
+  /**
+   * starts the process of reading messages from Iot Hub by creating an Event Hubs connection string
+   * @async
+   * @param {any} startReadMessageCallback - callback function
+   */
+  async startReadMessage(startReadMessageCallback) 
+  {
+    try 
+    {
+      const eventHubConnectionString = await convertIotHubToEventHubsConnectionString(this.iotHubConnectionString);
+      this.consumerClient = new EventHubConsumerClient(this.consumerGroup, eventHubConnectionString);
```

Comment:
right now if start gets called multiple times we create a new client and subs without closing the old ones. this is something we need to protect from, either by returning with no-op if already set or closing the old ones before starting new ones. the later is probably better as the callback could be different.